src/main/resources/application.yaml

```diff
@@ -113,7 +113,7 @@ idam:
     totp_secret: ${IDAM_KEY_CFT_HEARING_SERVICE:AAAAAAAAAAAAAAAA}
     url: ${S2S_URL:http://localhost:4502}
   s2s-authorised:
-    services: ${CFT_HEARING_SERVICE_S2S_AUTHORISED_SERVICES:xui_webapp,hmc_hmi_inbound_adapter,sscs}
+    services: ${CFT_HEARING_SERVICE_S2S_AUTHORISED_SERVICES:xui_webapp,hmc_hmi_inbound_adapter,ccd_data,ccd_gw,sscs}
```

Comment:
have these services only been added for befta?

Reply:
Seems as though this was added during a merge from master - but comparing against master now it doesnt look like its there - should i remove these? i dont think theyre for the befta

Reply:
`    services: ${CFT_HEARING_SERVICE_S2S_AUTHORISED_SERVICES:xui_webapp,hmc_hmi_inbound_adapter,sscs}`

Please can we remove